src/payment/strategies/braintree/braintree.ts

```diff
@@ -299,14 +300,41 @@ export interface BraintreeShippingAddress extends BraintreeAddress {
     recipientName: string;
 }
 
+export interface VenmoError {
+    code: string;
+    details: string;
+    message: string;
+    type: string;
+    name: string;
+}
+
+export interface VenmoInstance {
+    tokenize?(callback: Tokenize): void;
+    isBrowserSupported(): boolean;
+    create(arg0: VenmoConfig, arg1: VenmoCreateCallback ): void;
```

Comment:
Can we please give some meaningful names to args like config, callback?

Reply:
renamed